Modules/Core/Common/include/itkMacro.h

```diff
@@ -394,6 +394,15 @@ namespace itk
     static_assert(false, "Replace deprecated ITK_DISALLOW_COPY_AND_ASSIGN with modern ITK_DISALLOW_COPY_AND_MOVE")
 #endif
 
+#if __cplusplus < 202002L
+#  define ITK_DEFINE_INEQUAL_OPERATOR_MEMBER_FUNCTION(TypeName)                                                        \
```

Comment:
I think the proper word is `unequal`, not `inequal`. See https://www.merriam-webster.com/words-at-play/unequal-vs-inequality-negative-prefix-usage.

Reply:
FWIW, https://en.cppreference.com/w/cpp/language/operator_comparison and https://www.cplusplus.com/doc/tutorial/operators/ say the operator's name is "not equal to".

Reply:
@Leengit Thanks, but sorry, Dženan was first 😸 By the way, I also like `NONEQUAL` and `NON_EQUAL`. Maybe we should make the macro name user-configurable  ;-)

Reply:
@dzenanz Please check the new name   :-)